controller/internal/enforcer/applicationproxy/http/http.go

```diff
@@ -261,6 +294,7 @@ func (p *Config) RunNetworkServer(ctx context.Context, l net.Listener, encrypted
 		DialContext:         networkDialerWithContext,
 		MaxIdleConnsPerHost: 2000,
 		MaxIdleConns:        2000,
+		ForceAttemptHTTP2:   true,
```

Comment:
the force is done for the golang issue: https://github.com/golang/go/issues/21336